tutorials/introduction-gee-python-api-guiattard/tutorial.ipynb

```diff
@@ -0,0 +1,1244 @@
+{
```

Comment:
<p>Agree. The paragraph now reads as follow:</p><p> </p><blockquote>Printing&nbsp;the&nbsp;first&nbsp;5&nbsp;lines&nbsp;of&nbsp;the&nbsp;result&nbsp;shows&nbsp;that&nbsp;we&nbsp;now&nbsp;have&nbsp;arrays&nbsp;full&nbsp;of&nbsp;data.&nbsp;As&nbsp;we can&nbsp;see,&nbsp;several&nbsp;<em>None</em>&nbsp;values&nbsp;appear&nbsp;in&nbsp;the&nbsp;<em>LST_Day_1km</em>&nbsp;column.&nbsp;The&nbsp;associated quality&nbsp;indicator&nbsp;<em>QC_Day</em>&nbsp;indicates&nbsp;a&nbsp;value&nbsp;of&nbsp;<em>2</em>&nbsp;meaning&nbsp;that&nbsp;the&nbsp;LST&nbsp;is&nbsp;not&nbsp;calculated&nbsp;because&nbsp;of&nbsp;cloud&nbsp;effects.</blockquote>